lane_detect_main.cpp

```diff
@@ -0,0 +1,8 @@
+#include "lane_marker_detect.hpp"

```

Comment:
Let's find a way to merge our main.cpp's together. For now, work on this branch as your code is kinda isolated from the rest of the code